ourhexenv.py

```diff
@@ -74,20 +74,35 @@ def reset(self):
             self.window.fill(self.BACKGROUND)
             pygame.display.flip()
         
+    def _sparse_reward_on_win(self):
```

Comment:
Please take a look at the code for #19 that has merged. We are implementing rewards as static dictionaries instead of functions.

Reply:
@Mayukh747, this is to improve readability, I don't see what the problem is here? This method improves readability and utilizes the same static dictionary implementation. Can you please elaborate as to why you suggested this change?

Reply:
Sure, 

So the approach that we are taking with the reward mapping is to conditionally initialize it based on the sparse_flag [in the __init__() function] (https://github.com/sjsu-interconnect/ourhexgame/blob/8d933980ce666209657f2d498dcfc6b75a0cf620/ourhexenv.py#L88).

Since the reward_mapping dictionary is already assigned the correct rewards, there's no need to write additional logic to decide what the reward values will be,[just use the dictionary as is](https://github.com/sjsu-interconnect/ourhexgame/blob/8d933980ce666209657f2d498dcfc6b75a0cf620/ourhexenv.py#L193).

Lmk if that doesn't make sense.

Reply:
Yes I see what you mean, thank you for the explanation. I see this change was pushed to the main branch, have resolved conflicts, should be good to go.